### PR TITLE
Fix for 1008

### PR DESCRIPTION
**Pull request by Shruthi-1MN**
**What this PR does / why we need it**:
Fix for 1008, fileshare name supports up to 255 characters name length

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/api/util/db.go [hunk 1]
```diff
@@ -176,8 +176,8 @@ func CreateFileShareDBEntry(ctx *c.Context, in *model.FileShareSpec) (*model.Fil
 		log.Error(errMsg)
 		return nil, errors.New(errMsg)
 	}
-	if len(in.Name) > 128 {
-		errMsg := fmt.Sprintf("fileshare name length should not more than 128 characters. current length is : %d", len(in.Name))
+	if len(in.Name) > 255 {
+		errMsg := fmt.Sprintf("fileshare name length should not more than 255 characters. current length is : %d", len(in.Name))
```
Comment (kumarashit) [inline pkg/api/util/db.go]:
```suggestion
		errMsg := fmt.Sprintf("fileshare name length should not be more than 255 characters. Provided length is : %d", len(in.Name))
```

File: pkg/api/util/db_test.go [hunk 2]
```diff
@@ -533,14 +533,14 @@ func TestCreateFileShareDBEntry(t *testing.T) {
...
+	t.Run("File share name length more than 255 characters are not allowed", func(t *testing.T) {
+		in.Size, in.Name, in.ProfileId = int64(1),"kjxdpvfuenecpyhbxdumwibipjkigcjkkkhkhkhkkkkkkkkkkkkkkkkkkkkkkkkkknnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkkykabuhghjghjgjgjgjgjghhkkkkkkkkkkkkkkklsccdesnkgxnjnhfjusigceorogdgjlyciemscgvncerucokfekdiviuyplbly", "b3585ebe-c42c-120g-b28e-f373746a71ca"
 		mockClient := new(dbtest.Client)
 		mockClient.On("CreateFileShare", context.NewAdminContext(), in).Return(&SampleFileShares[0], nil)
 		db.C = mockClient
 
 		_, err := CreateFileShareDBEntry(context.NewAdminContext(), in)
-		expectedError := "fileshare name length should not more than 128 characters. current length is : "+strconv.Itoa(len(in.Name))
+		expectedError := "fileshare name length should not more than 255 characters. current length is : "+strconv.Itoa(len(in.Name))
```
Comment (kumarashit) [inline pkg/api/util/db_test.go]: Same as above

**Comment by kumarashit** [comment]
@Shruthi-1MN Please raise it for development branch

**Comment by Shruthi-1MN** [comment]
This is a duplicated of merged PR #1077 . So closing this PR.

**Pull request closed without merging**